source/usr/lib/mcvirt/parser.py

```diff
@@ -203,6 +203,8 @@ def __init__(self, verbose=True):
                                         dest='hard_disk_driver', type=str,
                                         help='Driver for hard disk',
                                         default=None)
+        self.create_parser.add_argument('--set-win10', help='Add Windows 10 CPU flags',
```

Comment:
As suggested previously, this should probably be a generic windows flag, instead of windows10 specific